ccd-definition/CaseEvent/Camunda/CaseEvents-CUI-CP-nonprod.json

```diff
@@ -11,5 +11,31 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "NOTIFY_CLAIMANT_AMEND_RESTITCH_BUNDLE",
+    "Name": "Notify Claimant",
```

Comment:
Could you make the name more descriptive?
Like **Notify Claimant Amend Bundle** maybe?